.github/workflows/publish-python-package.yml

```diff
@@ -6,7 +6,8 @@ name: Publish Python Package
 
 on:
   release:
-    types: [created]
```

Comment:
I think we'd want to maintain `created` on here ... just want to add the functionality of a limit of what branch on which the workflow can run